modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPackageTest.java

```diff
@@ -1123,4 +1120,82 @@ private void createBugSites() throws Exception {
         geopkg.add(entry, shp.getFeatureSource(), null);
         assertTableExists("bugsites");
     }
+
+    // if the FC already is XY, then forceXY does nothing (should return same FC)
+    @Test
+    public void testForceXY_alreadyXY() throws Exception {
+
+        String wkt_xy =
+                "GEOGCS[\"WGS 84\", \n"
+                        + "  DATUM[\"World Geodetic System 1984\", \n"
+                        + "    SPHEROID[\"WGS 84\", 6378137.0, 298.257223563, AUTHORITY[\"EPSG\",\"7030\"]], \n"
+                        + "    AUTHORITY[\"EPSG\",\"6326\"]], \n"
+                        + "  PRIMEM[\"Greenwich\", 0.0, AUTHORITY[\"EPSG\",\"8901\"]], \n"
+                        + "  UNIT[\"degree\", 0.017453292519943295], \n"
+                        + "  AXIS[\"Geodetic latitude\", EAST], \n"
+                        + "  AXIS[\"Geodetic longitude\", NORTH], \n"
+                        + "  AUTHORITY[\"EPSG\",\"4326\"]]";
```

Comment:
You may be able to directly use the `DefaultGeographicCRS.WGS84`; it is a constant defined with the above axis order.

Reply:
I'd like to be explicit (and obvious) and be consistent with the other test case, below.

Reply:
I added a comment to make this more obvious.